src/index.js

```diff
@@ -3,9 +3,10 @@ const path = require("path"),
 	worker = path.join(__dirname, "worker.js"),
 	events = /^(error|message)$/,
 	defaultPorts = {inspect: 9229, debug: 5858};
+let range = {min: 1, max: 300};
 
 class Worker {
-	constructor (arg, args = undefined, options = {cwd: process.cwd()}) {
+	constructor (arg, args = [], options = {cwd: process.cwd()}) {
```

Comment:
Why this? It gets passed without a further change, which should trigger the default handling within the function if it renames `undefined`.